app/services/idv/session.rb

```diff
@@ -136,7 +135,6 @@ def applicant_params_ascii
     def profile_maker
       @_profile_maker ||= Idv::ProfileMaker.new(
         applicant: applicant_params,
-        normalized_applicant: normalized_applicant_params,
         phone_confirmed: vendor_phone_confirmation || false,
         user: current_user
       )
```

Comment:
Should `applicant_params_ascii` be removed from this file, and `vendor_params` updated to use `applicant_params` instead of `applicant_params_ascii`?

Reply:
hmmm, i'm not 100% on the requirements here, but I assumed we were limiting the strings to ascii for a reason, possibly because the json en/decoding required to pass the data to the background jobs can have issues with certain utf8 characters. I can find smatterings of these issues on the web, such as https://github.com/JoshCheek/seeing_is_believing/issues/46

Reply:
The only reason for converting parameters to ASCII was to please Equifax. That code was introduced in this PR: https://github.com/18F/identity-idp/pull/1137

If we're worried that people might try to send invalid UTF-8 bytes during IdV, I think that's a separate issue.

Reply:
We can address this later, though.